Makefile

```diff
@@ -106,11 +101,10 @@ app :
 	-http localhost:30080/version
```

Comment:
Got the following error when checking the endpoint. Include pod status
```bash
NAME READY STATUS RESTARTS AGE
jumpbox 1/1 Running 0 21m
ngsa-memory 1/1 Running 0 4s
webv 1/1 Running 0 2s
# display the app version
http localhost:30080/version



http: error: ConnectionError: ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')) while doing a GET request to URL: http://localhost:30080/version



make: [Makefile:96: app] Error 1 (ignored)
```

pods looked good, and got the expected response when I ran http localhost:30080/version manually

Reply:
that is likely a timing error - I'll put a wait for pods into the makefile

Reply:
I ran into the same error. A readiness probe pointing to "/healthz" fixed it for me locally. I didn't add it directly to this PR in case probes were left out intentionally.

```yaml
# in deploy/ngsa-local/ngsa-memory.yaml
...
    ports:
    - containerPort: 8080
    readinessProbe:
      initialDelaySeconds: 5
      httpGet:
        path: /healthz
        port: 8080
```

Reply:
Also if someone could add the issue and work with Donovan on the dapr run issue - it's consistently works the first time for me then fails on 2+ with the kestrel port in use error. Something appears to be hanging around.

Reply:
I converted the PR to draft to fix these bugs

Reply:
Issues have been created here
https://github.com/retaildevcrews/k3d-codespaces-template/issues/27
https://github.com/retaildevcrews/k3d-codespaces-template/issues/28